docs/scarpet/api/Auxiliary.md

```diff
@@ -42,89 +56,105 @@ Throws `unknown_particle` if particle doesn't exist.
-     * `to` - other end of the line, same format as `from`
-     
- * `'box'` - draws a box with corners in specified points
-   * Required attributes:
-     * `from` - triple coordinates, entity, or block value indicating one corner of the box
-     * `to` - other corner, same format as `from`
-
- * `'sphere'`:
-   * Required attributes:
-     * `center` - center of the sphere
-     * `radius` - radius of the sphere
-   * Optional attributes:
-     * `level` - level of details, or grid size. The more the denser your sphere. Default level of 0, means that the
-      level of detail will be selected automatically based on radius.
-      
- * `'cylinder'`:
-   * Required attributes:
-     * `center` - center of the base
-     * `radius` - radius of the base circle
-   * Optional attributes:
-     * `axis` - cylinder direction, one of `'x'`, `'y'`, `'z'` defaults to `'y'`
-     * `height` - height of the cyllinder, defaults to `0`, so flat disk. Can be negative.
-     * `level` - level of details, see `'sphere'`.
-
+* `'line'` - draws a straight line between two points.
+    * Required attributes:
+        * `from` - triple coordinates, entity, or block value indicating one end of the line
+        * `to` - other end of the line, same format as `from`
+
+* `'label'` - draws a text in the world. Default `line` attribute controls main font color.
+  `fill` controls the color of the background.
+    * Required attributes:
+        * `pos` - position
+        * `text` - string or formatted text to display
+    * Optional attributes
+        * `value` - string or formatted text to display instead of the main `text`. `value` unlike `text`
+          is not used to determine uniqueness of the drawn text so can be used to
+          display smoothly dynamic elements where value of an element is constantly
+          changing and updates to it are being sent from the server.
+        * `size` - float. Default font size is 10.
+        * `facing` - text direction, where its facing. Possible options are: `player` (default, text
+          always rotates to face the player), `north`, `south`, `east`, `west`, `up`, `down`
+        * `doublesided` - if `true` it will make the text visible from the back as well. Default is `false` (1.16+)
+        * `align` - text alignment with regards to `pos`. Default is `center` (displayed text is
+          centered with respect to `pos`), `left` (`pos` indicates beginning of text), and `right` (`pos`
+          indicates the end of text).
+        * `tilt`, `lean`, `turn` - additional rotations of the text on the canvas along all three axis
+        * `indent`, `height`, `raise` - offsets for text rendering on X (`indent`), Y (`height`), and Z axis (`raise`)
+          with regards to the plane of the text. One unit of these corresponds to 1 line spacing, which
+          can be used to display multiple lines of text bound to the same `pos`
```

Comment:
Label is already below, so this is pretty much a copy of the one below. One should be removed.